mcm/rest_api/RequestActions.py

```diff
@@ -1639,22 +1641,37 @@ def is_in_chain(r):
                 if len(mcm_r['member_of_chain']) == 0: continue
                 # to get a remind only on request that are being necessary to move forward : being the request being processed in at least a chain.
                 on_going = False
+                yield '.'
                 for in_chain in mcm_r['member_of_chain']:
                     mcm_cr = chained_request( crdb.get( in_chain ) )
-                    if mcm_cr.get_attribute('chain')[mcm_cr.get_attribute('step')] == rid:
-                        on_going = True
-                        break
+                    try:
+                        if mcm_cr.get_attribute('chain')[mcm_cr.get_attribute('step')] == rid:
+                            on_going = True
+                            break
+                    except Exception as e:
```

Comment:
add a error log to save the chain_req id and request id. This actually happens when writing to DB fails, and means that manual editing is needed.